packages/components/src/Banner/Banner.tsx

```diff
@@ -103,17 +89,10 @@ export function Banner({
       </div>
 
       {dismissible && (
-        <button
-          type="button"
-          className={styles.closeButton}
+        <ButtonDismiss
```

Comment:
Since icon is no longer changing color, we can use the standard ButtonDismiss here now.